trino-s3-proxy/src/main/java/io/trino/s3/proxy/server/TrinoS3ProxyServerModule.java

```diff
@@ -43,6 +48,8 @@ public final void configure(Binder binder)
         binder.bind(TrinoS3ProxyClient.class).in(Scopes.SINGLETON);
 
         moduleSpecificBinding(binder);
+
+        OptionalBinder.newOptionalBinder(binder, StsController.class).setDefault().toInstance(StsController.DEFAULT);
```

Comment:
is there any advantage to using an `OptionalBinder` if we provide just a default? is this done for future-proofing?

Reply:
Yeah, it doesn't improve it. I'll change it.